buildtime-analysis/visualize_builds.py

```
@@ -0,0 +1,338 @@
+        ongoing_builds = set()
+        for event in sorted_events:
+            if event['event_type'] == 'build_start':
+                ongoing_builds.add(event['package'])
+            elif event['event_type'] == 'build_end':
+                ongoing_builds.discard(event['package'])
+
+            time_points.append(event['timestamp'])
+            concurrent_counts.append(len(ongoing_builds))
+
+        # Create the plot
+        fig, (ax1, ax2) = plt.subplots(2, 1, figsize=(16, 12))
```

Ruff (0.15.2)

[warning] Unpacked variable `fig` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)